.readthedocs.yml

```diff
@@ -1,26 +1,23 @@
 version: 2
 
+build:
+  os: ubuntu-20.04
+  apt_packages:
+    - graphviz
+  tools:
+    python: "3.10"
```

Comment:
What about
```
build:
   image: latest
```
?

Reply:
That's the legacy build config (https://docs.readthedocs.io/en/stable/config-file/v2.html#legacy-build-specification).  This PR updates to the new config.